### ISSUE TITLE
Protótipo do Projeto

### ISSUE THREAD

**Issue by mendesiasmin** [fga-eps-mds/2018.2-IndicaAi#142]

**Comment by GabrielDVpereira** [comment]
@mendesiasmin @mg5376 o protótipo que tá lá no Git é aquele que você mostrou ontem já?

**Comment by mendesiasmin** [comment]
@GabrielDVpereira Vê a data do último commit do @mg5376 na branch dele, se for de ontem deve estar atualizado

**Comment by matheusgomesf** [comment]
@mendesiasmin @Gabriel299 ainda não dei push na última versão, vou fazer algumas últimas mudanças e vou subir

**Comment by mendesiasmin** [comment]
Galera, o pessoal do App ta conversando de não fazer menu hamburguer porque pode ficar confuso para o usuário cada app ter um menu hamburguer diferente. A ideia que eles deram é fazer algo parecido com a google play (https://www.xda-developers.com/files/2017/11/Screenshot_20171105-203313-576x1024.jpg), que tem essas tabs em cima para o usuário.

**Comment by mendesiasmin** [comment]
@GabrielDVpereira Sim, vai ter que tranformar ele em algo parecido com o da play store

**Comment by GabrielDVpereira** [comment]
@mendesiasmin Já como vai ser essa barra que seria a superior? Seria aquela que fica em baixo?

**Comment by mendesiasmin** [comment]
@GabrielDVpereira Não entendi Gabriel
